build.sbt

```diff
@@ -19,8 +19,20 @@ lazy val root = Project(libName, file("."))
       "com.vladsch.flexmark" %  "flexmark-all" % "0.35.10" % "test"
     ),
     scriptedLaunchOpts := {
+      val homeDir = sys.props.get("jenkins.home")
+        .orElse(sys.props.get("user.home"))
+        .getOrElse("")
       scriptedLaunchOpts.value ++
-        Seq("-Xmx1024M", "-Dplugin.version=" + version.value)
+        Seq(
+          "-Xmx1024M",
+          "-Dplugin.version=" + version.value,
+          s"-Dsbt.override.build.repos=${sys.props.getOrElse("sbt.override.build.repos", "false")}",
+          // s"-Dsbt.global.base=$sbtHome/.sbt",

```

Comment:
Do we want to keep the commented out lines of code?

Reply:
I think that kind of goes with the comment below, like "ideally we'd just set this, but ...", so I left it in there